tests/layout/test_layout_control_mode.py

```diff
@@ -93,16 +92,11 @@ async def test_callback_signal_control_prohibit_control(qtbot, widget):
 @pytest.mark.asyncio
 async def test_switch_force_balance_system(qtbot, widget_async):
 
-    controller = widget_async.model.controller
-    await controller.write_command_to_server(
-        "start", controller_state_expected=salobj.State.DISABLED
-    )
-    await controller.write_command_to_server(
-        "enable", controller_state_expected=salobj.State.ENABLED
-    )
+    await widget_async.model.enter_diagnostic()
+    await widget_async.model.enter_enable()
 
     # Sleep so the event loop can access CPU to handle the signal
-    await asyncio.sleep(1)
+    await asyncio.sleep(15)
```

Comment:
Does it really need to be 15 seconds? enter_enable should be much faster, shouldn't it?

Reply:
This is for the event loop to handle the incoming telemetry and events, which takes some time. Thanks!